openedx/core/djangoapps/agreements/models.py

```diff
@@ -22,3 +22,15 @@ class IntegritySignature(TimeStampedModel):
     class Meta:
         app_label = 'agreements'
         unique_together = ('user', 'course_key')
+
+
+class LTIPIITool(models.Model):
+    """
+    This model stores the relationship between a course and the LTI tools in the course that share PII.
+    """
+    course_key = CourseKeyField(max_length=255, unique=True, db_index=True)
+    lti_tools = models.JSONField()
+    lti_tools_hash = models.IntegerField()
+
+    class Meta:
+        app_label = 'agreements'
```

Comment:
I don't think we need this, because `agreements` is in the `INSTALLED_APPS` setting. You can see that [here](https://github.com/openedx/edx-platform/blob/96699d577c02e1a45225c26a8959f3e14e92aa5c/lms/envs/common.py#L3250-L3251). Could you remove the meta class and see that everything is still fine?

Reply:
I'm getting this error from testing:

> ERROR openedx/core/djangoapps/courseware_api/tests/test_views.py - RuntimeError: Model class openedx.core.djangoapps.agreements.models.LTIPIITool doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.

Reply:
Thanks for trying!